app/serializers/rent_serializer.rb

```diff
@@ -0,0 +1,14 @@
+class RentSerializer < ActiveModel::Serializer
+  attributes :id, :from, :to
+
+  has_one :book, serializer: BookSerializer
+  has_one :user, serializer: UserSerializer
+
+  def from
```

Comment:
:disappointed: sorry, we are going to use it at `starting_date` and `ending_date`